### DISCUSSION TITLE
Syntax error when executing start.bat

### DISCUSSION THREAD

**Discussion by k1nji** [Discord-OTP-Forcer/Discord-OTP-Forcer#81] [Community Support] [answered]
I am encountering a syntax error when executing start.bat
Here is the screenshot of the error
![SyntaxError](https://github.com/Derpitron/Discord-OTP-Forcer/assets/147069764/2335f9e3-a6e8-4952-9c52-000f815d58b3)

**Comment by Inbydev** [comment]
@k1nji could you try with the latest commit?

https://github.com/Derpitron/Discord-OTP-Forcer/archive/refs/heads/main.zip

And follow the tutorial:

https://github.com/Derpitron/Discord-OTP-Forcer/wiki/How-to-setup-and-use

**Comment by Inbydev** [comment, marked as answer]
This probably is an Issue of the past, the program got a lot of updates since then, so re open this discussion if you encounter the same issue with the latest release!

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta

And follow the tutorial:

https://discord-otp-forcer.codeberg.page/en/
